CHANGELOG.md

```diff
@@ -3,6 +3,153 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+## 7.0.0-alpha.9
+_Jan 18, 2024_
+
+We'd like to offer a big thanks to the 11 contributors who made this release possible. Here are some highlights ✨:
+
+- 🎁 Allow to filter non-filterable columns programmatically (#11538) @MBilalShafi
+- 🚀 Make `PickersTextField` and its dependencies public (#11581) @flaviendelangle
+- 🌍 Add Croatian (hr-HR), Portuguese (pt-PT), and Chinese (Hong Kong) (zh-HK) locales (#11668) on the Data Grid @BCaspari
+- 🐞 Bugfixes
+
+### Breaking changes
+
```

Comment:
was wondering if we should add a `#### Data Grid` here, but since there are only grid BCs this did not make much sense

Reply:
The grid breaking changes should be an h4 **inside** the grid section :+1: